pkg/plugins/secrets/k8s/store.go

```diff
@@ -302,9 +301,6 @@ func (c *SimpleConverter) ToCoreList(in *kube_core.SecretList, out core_model.Re
 			}
 			secOut.Items[i] = r
 		}
-		sort.SliceStable(secOut.Items, func(i, j int) bool {
```

Comment:
to get this right, the previous store could have partial response here and this is how it was not sorted?

Reply:
Not sure if that could be the case. I think we may have overlooked sorting in this situation due to the delegation model, where tasks are delegated from one entity to another. The secret store wasn't positioned behind the pagination store, which ensures that results are sorted.